index.js

```diff
@@ -467,6 +474,9 @@ class MillerColumnsElement extends HTMLElement {
         } else if (showNarrow) {
           item.classList.add(narrowClass)
         }
+        if (columnsToShow.length === 0) {
```

Comment:
This looks like it's covering an edge case - probably best moved to its own commit.

Reply:
Extracted this into its own commit.

Reply:
Thanks - that makes perfect sense to me now.